snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
Very strange...
`python3
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.linux_distribution()[0]
'LinuxMint'
>>> 
`